Fixed conjob and added database instructions

november781 opened the pull request:
Cron needed its logging directory made in the image (decided to add that to the Dockerfile and let it be in the image instead of doing it in the entrypoint script)

The default_cronjob needed to have sudo so it could make a local php call instead of root

Since it looks like the static database still needs to be imported, I added the .zip and instructions for it. Solves #12

november781 committed:
Cron needed some fixes

november781 committed:
Added static databse and instructions for linking

november781 committed:
clarification

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
-3. Restart your container with `SETUP=False`
-4. You're live!
+2. Create the databases using the database controls in the setup page.
+3. [Import static databse.](#Importing-static-databse)
```
typo here

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+6. You're live!
+
+# Importing staic database
+1. Download the static database.
```
Should just specify command: `wget https://github.com/exodus4d/pathfinder/raw/master/export/sql/eve_universe.sql.zip`

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+
+# Importing staic database
+1. Download the static database.
+2. Unzip `eve_universe.sql.zip`.
```
Should add command: `unzip eve_universe.sql.zip`

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+1. Download the static database.
+2. Unzip `eve_universe.sql.zip`.
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
+4. Enter the db container `docer-compose exec db /bin/bash`.
```
Typo here... `docker-compose exec db /bin/bash`

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+# Importing staic database
+1. Download the static database.
+2. Unzip `eve_universe.sql.zip`.
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
```
Typo here... `database`

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+2. Unzip `eve_universe.sql.zip`.
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
+4. Enter the db container `docer-compose exec db /bin/bash`.
+5. Make sure you have completed step 2 in Setup.
```
This line isn't needed imo, they're following steps

porowns commented on README.md:
```diff
@@ -7,8 +7,22 @@ Dockerfile for running [Pathfinder](https://github.com/exodus4d/pathfinder), the
+3. Copy the databse into the db container `docker cp eve_universe.sql {full container name: eg. pathfinder-docker_db_1}:/eve_universe.sql`.
+4. Enter the db container `docer-compose exec db /bin/bash`.
+5. Make sure you have completed step 2 in Setup.
+6. Link static databse `mysql -u root -p eve_universe< eve_universe.sql`.
```
Typo

november781 committed:
Fix typos

woops, that is why I'm not an English major (or front-end dev)

november781 reviewed (commented):
Fixed typos

november781 committed:
added commands as requested

november781 reviewed (commented):
added those commands as requested

november781 committed:
removed unneeded line

november781 committed:
updated to more user friendly copy-paste commands

november781 commented:
all changes should be up-to-date and ready for merge

november781 committed:
Don't need this file

since we are pulling from the main Pathfinder repo with wget, this file is unnecessary
